### PR TITLE
Removes Gompertz Model

### PR DESCRIPTION
**Pull request by zeroset**
We are refactoring the simulation and stick to velocity and GCFM
Model for the moment. Gompertz Model will be reintroduced later.

### CODE REVIEW

File: xsd/jps_ini_core.xsd [hunk 1]
```diff
@@ -428,7 +428,6 @@
                   <xs:attribute name="description" use="required">
                     <xs:simpleType>
                       <xs:restriction base="xs:string">
-                        <xs:enumeration value="gompertz" />
```
Comment (schroedtert) [inline xsd/jps_ini_core.xsd]: Isn’t the velocity model missing?

File: math/ForceModel.cpp [hunk 2]
```diff
@@ -21,7 +21,7 @@
  * along with JuPedSim. If not, see <http://www.gnu.org/licenses/>.
  *
  * \section Description
- * Implementation of the GCFM and the Gompertz model.
+ * Implementation of the GCFM.
```
Comment (schroedtert) [inline math/ForceModel.cpp]: Is this class needed? It is only used in ODESolver, which itself is not used anywhere (as far as I can see)

Comment (zeroset) [reply math/ForceModel.cpp]: I think you are right. I am cleaning up the models step by step. Will be removed in a following pr.

File: xsd/jps_ini_core.xsd (hunk 1 above)
Comment (zeroset) [reply xsd/jps_ini_core.xsd]: The whole xsd needs an update I think. Just wanted to remove gompertz...

**Pull request merged**